enterprise/api_client/discovery.py

```diff
@@ -165,6 +165,10 @@ def get_catalog_results(self, content_filter_query, query_params=None, traverse_
                     query_params,
                     traverse_pagination
                 )
+                LOGGER.info(
+                    'ENT-2489 | Response has size %d',
```

Comment:
it might be good for us to see which `content_filter_query` is associated with the response, could you add it to this message?

Reply:
👌